### PR TITLE
Created a personal file for jaelle

### PR DESCRIPTION
**Pull request by jaelle**
This feature adds the jaelle.md file into the `personal-files` folder.

### CODE REVIEW

**Comment by crichID** [comment]
:+1: Good job jaelle!

File: personal-files/jaelle.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Jaelle was here
```
Comment (githubteacher) [inline personal-files/jaelle.md]: Maybe add a bit more description about what you're working on

Comment (jaelle) [inline personal-files/jaelle.md]: @githubteacher I added a couple more details!

**Comment by mandy72** [comment]
Looks good Jaelle :+1:

**Pull request merged**